### ISSUE TITLE
Show list of elements of an overlay + add bi-directional interaction

### ISSUE THREAD

**Issue by peterdesmet** [trias-project/risk-maps#16]
It took me a while to notice that the name of the hovered overlay was shown on the right. It also disappears right away if you hover away. I think this could be improved by:

1. Showing all elements of an overlay on the side
2. Highlighting the element when it is hovered on the map **and vice versa**
3. Highlighting the element when it is clicked on the map **and vice versa**
4. Clicking an element (map or side) keeps it selected. This selection is undone by a) clicking it again, b) highlighting another area, c) clicking anywhere else on the map (including outside the full overlay)

<img width="1143" alt="Screenshot 2020-04-03 at 14 03 45" src="https://user-images.githubusercontent.com/600993/78358721-f3512a80-75b3-11ea-8658-4b0055d6daf2.png">

**Comment by SoVDH** [comment]
Same for the two entities of Fagnes-Famenne-Calestienne

**Comment by niconoe** [comment]
@peterdesmet : love the idea, will do that!

@SoVDH : does your comment applies to #15 instead?

**Comment by SoVDH** [comment]
Indeed, sorry for that ;-)

**Comment by peterdesmet** [comment]
Come to think of it, listing the elements on the side will give too much attention to the overlay (and take up a lot of space). Instead, I propose to show the selected element (current implementation) on hover **and click**, but also step 4 of the issue above, so it doesn't go away so fast.

**Comment by niconoe** [comment]
Sorry @peterdesmet, I'm not sure I still follow. Could you rephrase what will happen on hover and/or click, on the map and the list?

We can make a quick call or chat session to discuss this if easier/faster! thx

**Comment by peterdesmet** [comment]
Sure!

## Hover

Keep current implementation

## Click

- [x] Same behaviour as hover, but keep area selected when moving mouse away.
- [ ] Allow to deselect an area by a) clicking it again, b) highlighting another area, c) clicking anywhere else on the map (including outside the full overlay)
- [x] Remove current zoom in behaviour

Keep current implementation/style on the side, showing only the highlighted area

**Comment by niconoe** [comment]
@peterdesmet:

I now have "almost working" code for all this, but I'm considering (temporarily?) disabling the "click" part, because after trying it, the cost/benefit ratio seems not that great in practice:

Benefits: 
- Possibility for the use to keep a zone highlighted (but not sure of the practical usefulness)

Costs:
- I'm concerned that in some cases, the interactions are not that clear for the end user (example: "deselect by highlighting another area": the behaviour change if the mouse leave a clicked polygon to go to another polygon, or just to an empty part of the map)
- This is a bit messy to implement, so it adds quite a bit of code and complexity to the app, especially in combination with #15 and the refactoring that will be needed for #21 (data originating from the data cube).

Ok for you if I remove the click-related code for now, and we'll re-enable it later (once the codebase is a bit more stable) if necessary?

**Comment by peterdesmet** [comment]
That is fine by me, I would even suggest to label this issue as `wontfix` and close it. Most of the original suggestions in this issue (showing all regions) have been reverted anyway and the click part doesn't contribute much. Issue https://github.com/trias-project/risk-maps/issues/15 is a bit more bothersome.

**Comment by niconoe** [comment]
Perfect, I indeed do plan to tackle #15 soon!
